src/core/types.d.ts

```diff
@@ -47,6 +47,11 @@ type TwoFAConfig = {
   threshold: number
 }
 
+type LocaleConfig = {
```

Comment:
`src/core` is deprecated, the legacy code is living there and this directory will soon be removed. we're now using domain driven design. 

maybe you can create a `src/domain/localization` and add this type here. 

I think having some unit tests on this PR would also be useful.

Reply:
Understood, I've moved to `src/domain/localization` as requested.

I will push some unit tests.